lib/views/help/_contact_form.html.erb

```diff
@@ -0,0 +1,64 @@
+<%= form_for :contact,
+             :url => help_contact_path + '#wdtk',
+             :html => {:class => 'contact-form'} do |f| %>
+
+    <p class="contact-form__understand">
+        <%= f.check_box :understand, :required => true %>
+        <label for="contact_understand" class="form_label">
+            I understand that WhatDoTheyKnow is <strong>not</strong> run by the
+            government, and the WhatDoTheyKnow volunteers <strong>cannot</strong>
+            help me with personal matters relating to government services.
+        </label>
+    </p>
+
+    <% if not @user %>
+        <p>
+            <label class="form_label" for="contact_name">My name:</label>
+            <%= f.text_field :name, :size => 20 %>
+            (or <%= link_to "sign in", signin_url(:r => request.fullpath) %>)
```

Comment:
@lizconlan – This return path should include the "#wdtk" location fragment, so that when they return to the contact page after logging in, they're automatically taken down to the right section of the page.